openapi.yml

```diff
@@ -182,9 +197,9 @@ paths:
               schema:
                 $ref: '#/components/schemas/BinaryFile'
         403:
-          description: Forbidden
+          $ref: "#/components/responses/403Forbidden"
         404:
-          description: File not found
+          $ref: https://teamdigitale.github.io/openapi/0.0.5/definitions.yaml#/responses/404NotFound
```

Comment:
I suggest moving all `teamdigitale.github.io` refs to `components/responses` and reference responses from there.